src/lib/Zikula/Core/Controller/AbstractController.php

```diff
@@ -108,7 +108,7 @@ protected function decorateTranslator(array $parameters): array
      * Returns a NotFoundHttpException; this will result in a 404 response code.
      * Usage example: throw $this->createNotFoundException();
      */
-    public function createNotFoundException(string $message = 'Not Found', ?Throwable $previous = null): NotFoundHttpException
+    public function createNotFoundException(string $message = 'Access Denied.', \Throwable $previous = null): NotFoundHttpException
```

Comment:
Please change `Access Denied.` back to `Not Found` like shown at https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php#L291